api-server/src/common/models/user.json

```
@@ -120,6 +120,11 @@
       "description": "Camper has signed academic honesty policy",
       "default": false
     },
+    "needsModeration": {
+      "type": "boolean",
+      "description": "Camper has challenges needing to be moderated",
+      "default": false
```

```suggestion
      "default": false,
      "index": {
        "mongodb": {
          "unique": true,
          "background": true
        }
      }
```

Let's add this anyway for consistency and to be safe.